libraries/SITL/SIM_GPS.cpp

```diff
@@ -1171,4 +1160,40 @@ void GPS::update()
     }
 }
 
+/*
+  get delayed data by interpolation
+*/
+GPS::gps_data GPS::interpolate_data(const gps_data &d, uint32_t delay_ms)
```

Comment:
How will this work with `SIM_GPS_GLITCH`?  We're supposed to get very specific jumps with those parameters.